src/OpenAL/Extensions/OpenTK.OpenAL.Extensions.Creative.EFX/Presets/ReverbPresets.cs

```diff
@@ -19,8 +19,7 @@ public static class ReverbPresets
         /// <summary>
         /// A reverb preset (approximating a generic location).
         /// </summary>
-        public static readonly ReverbProperties Generic = new ReverbProperties
-        (
+        public static readonly ReverbProperties Generic = new ReverbProperties(
```

Comment:
```cs
LongMethodInvocations
(
    arg,
    anotherArg
)
```
This is how the code should be and is a rule we sadly can't configure in Rider/ReSharper. Revert.